### PR TITLE
Happy new year

### PR DESCRIPTION
**Pull request by pcarruscag**
2020 -> 2021

### CODE REVIEW

File: Common/include/geometry/meshreader/CSU2ASCIIMeshReaderFVM.hpp [hunk 1]
```diff
@@ -36,78 +36,78 @@
  * \author: T. Economon
  */
 class CSU2ASCIIMeshReaderFVM: public CMeshReaderFVM {
-  
+
 private:
-  
+
```
Comment (pcarruscag) [inline Common/include/geometry/meshreader/CSU2ASCIIMeshReaderFVM.hpp:41]: ooops I did it again and stripped all the trailing spaces...

File: SU2_CFD/include/fluid/CMutationTCLib.hpp [hunk 2]
```diff
@@ -141,4 +141,4 @@ class CMutationTCLib : public CNEMOGas {
   vector<su2double>& GetSpeciesFormationEnthalpy() final;
 
 };
-#endif
\ No newline at end of file
+#endif
```
Comment (TobiKattmann) [inline SU2_CFD/include/fluid/CMutationTCLib.hpp:144]: ...and added some newlines at the end 🐤

File: SU2_CFD/src/output/CFlowIncOutput.cpp [hunk 3]
```diff
@@ -41,7 +41,7 @@ CFlowIncOutput::CFlowIncOutput(CConfig *config, unsigned short nDim) : CFlowOutp
 
   streamwisePeriodic             = config->GetKind_Streamwise_Periodic();
   streamwisePeriodic_temperature = config->GetStreamwise_Periodic_Temperature();
-      
+
```
Comment (TobiKattmann) [inline SU2_CFD/src/output/CFlowIncOutput.cpp:44]: upsi

**Comment by TobiKattmann** [comment]
Well happy new year to you as well 🎉 🥳 🍾 
I guess you did a find and replace over the repo for 2012-2020 -> 2012-2021. I briefly scrolled over the diff in case we are really unlucky and somewhere for some reason used that somewhere else. Maybe you had a more elaborate method :)

When I saw the `+-2`'s at a merge I first thought I missed a release :D

**Pull request merged**